nengo_dl/tests/test_tensor_graph.py

```diff
@@ -367,6 +368,17 @@ def test_create_signals_views():
     assert graph.signals[sigs[0]].slices == graph.signals[sigs[2]].slices
     assert graph.signals[sigs[1]].slices == graph.signals[sigs[3]].slices
 
+    slice_sig = Signal(np.ones((10, 4, 3, 2)))[1:4]
+    slice_sig.trainable = False
+    slice_sig.minibatched = True
+    slice_sig.base.trainable = False
+    slice_sig.base.minibatched = True
+    plan = [(dummies.Op(reads=[slice_sig]),)]
+    graph = dummies.TensorGraph(plan, tf.float32, 10)
+    graph.create_signals([slice_sig.base])
+    assert graph.signals[slice_sig].key == graph.signals[slice_sig.base].key
+    assert graph.signals[slice_sig].slices == ((1, 4),)
```

Comment:
Could the base signal not be ones, and we check the values of the sliced signal here? Also, could the slice have a `step` to test a bit more functionality?

Reply:
The value of the signal doesn't actually matter here, since it's not doing anything with the underlying value. It's just creating a `TensorSignal`, which abstractly represents a view on some other data (which hasn't been created here). But reading from sliced `TensorSignals` is tested (see `test_signal_dict_gather`), so as long as we're testing that the slices are created correctly here that should cover everything. I did add a `step` to the slice though to make sure that works properly.